### DISCUSSION TITLE
Error: unknown configuration variable 'advanced-telemetry'

### DISCUSSION THREAD

**Discussion by nanhuichen** [airsdk/Adobe-Runtime-Support#3233] [Q&A] [answered]
Hi,

I use Scout to profile my app, but I can't use 'advanced-telemetry'.
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/13010645/27b3ced8-902d-47a0-9211-5154da86e28a)

Can't 'advanced-telemetry' available now?
![image](https://github.com/airsdk/Adobe-Runtime-Support/assets/13010645/9cefc418-4f77-4e44-8b1a-07cb1104ef5e)

Thanks

environment:
Windows 10
FLEX_AIR_SDK_50.2.5

**Comment by Fancy2209** [comment, marked as answer]
can you try using mxmlc from the AIR SDK directly?

**Reply by nanhuichen** [reply]
Thank you for answer my stupid question, it is solved now when I used mxmlc from the AIR SDK  directly.

That mxmlc might point to mxmlc of Apache Royal.
